### DISCUSSION TITLE
Interesting observation when discharging while plugged in.

### DISCUSSION THREAD

**Discussion by JaiGupta13** [rurza/BatFi#76] [General]
When plugged to USB-C power, but SOC is above 80% so battery is draining, BatFi reports that 0.2-0.3W are coming from the power adapter. But according to a USB power meter, the voltage is negotiated to 20V but the current is 0.00A and there is no arrow symbol indicating that power is flowing. Probably not a bug, just something I noticed.

**Comment by holocronology** [comment]
Interesting. I'm seeing this same low power from USB-C/Thunderbolt connection from dock. 
![image](https://github.com/user-attachments/assets/0590d7f1-3cd7-49a5-ad9f-c35bb26388cb)
